config/manager/manager.yaml

```diff
@@ -19,6 +19,8 @@ spec:
   replicas: 1
   template:
     metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
```

Comment:
Oh my god  Does this fix seccomp issue?

Reply:
No, sorry. This sets the default for `kubectl logs` so that you don't have to specify the container and it defaults to the `manager` rather than `kube-rbac-proxy`.